.github/CODEOWNERS

```diff
@@ -79,7 +86,7 @@ runtime/centrifuge/* @branan @mikiquantum @mustermeiszer @NunoAlexandre @offerij
 runtime/integration-tests/* @mustermeiszer @NunoAlexandre
```

Comment:
I would like to be added here as well after spending quite some time on connectors integration tests.
```suggestion
runtime/integration-tests/* @mustermeiszer @NunoAlexandre @wischli 
```

Reply:
@cdamian should also be there